src/types.d.ts

```diff
@@ -54,3 +54,52 @@ export type RuleEvent = {
 	time: number;
 	internalLog?: string;
 }
+
```

Comment:
Not new definitions. Moved them over from the original ESLintEngine since they are reused between modules now.